addons/l10n_cl/views/report_invoice.xml

```diff
@@ -156,12 +156,31 @@
             <t t-set="l10n_cl_values" t-value="line._l10n_cl_prices_and_taxes()"/>
         </t>
 
+        <xpath expr="//span[@t-field='line.price_unit']" position="before">
+            <t t-set="line_amounts" t-value="line._l10n_cl_get_line_amounts()"/>
+            <t t-set="all_taxes" t-value="'all_taxes'"/>
+            <t t-if="'second_currency' in line_amounts" t-set="line_second_currency_round" t-value="line_amounts['second_currency']['round_currency']"/>
+        </xpath>
+
         <xpath expr="//span[@t-field='line.price_unit']" position="attributes">
             <attribute name="t-field"></attribute>
-            <attribute name="t-esc">line.price_unit</attribute>
-            <attribute name="t-options">{"widget": "monetary", "display_currency": o.currency_id}</attribute>
+            <attribute t-if="not o.l10n_latam_document_type_id._is_doc_type_electronic_ticket()" name="t-esc">line_amounts['price_item_document']</attribute>
+            <attribute name="t-options">{"widget": "monetary", "display_currency": line_amounts['main_currency']}</attribute>
         </xpath>
 
+        <th name="th_price_unit" t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}" position="after">
+            <th name="th_price_unit" t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
+                <span>Disc.%</span>
+            </th>
+        </th>
+
+        <td t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}" position="after">
+            <td t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
+                <span class="text-nowrap" t-esc="line_amounts['total_discount']" t-options="{'widget': 'monetary', 'display_currency': line_amounts['main_currency']}"/>
```

Comment:
comply with discount in amount and not in %

<img width="769" alt="Captura de pantalla 2023-10-02 a la(s) 17 26 25" src="https://github.com/odoo/odoo/assets/382893/9befb974-2621-4cae-b729-6c7e99cb8831">